libraries/AP_Mount/AP_Mount_Tusuav.cpp

```diff
@@ -0,0 +1,1109 @@
+#include "AP_Mount_Tusuav.h"
+#if HAL_MOUNT_TUSUAV_ENABLED
+#include <AP_Param/AP_Param.h>
+#include <AP_HAL/AP_HAL.h>
+#include <AP_AHRS/AP_AHRS.h>
+#include <AP_GPS/AP_GPS.h>
+#include <AP_RTC/AP_RTC.h>
+#include <GCS_MAVLink/GCS.h>
+#include <GCS_MAVLink/include/mavlink/v2.0/checksum.h>
+
+extern const AP_HAL::HAL& hal;
+
+#define AP_MOUNT_TUSUAV_HEADER1       0x5A     // first header byte
```

Comment:
Consider using typed variables rather than defines in here

Reply:
Thank you for the suggestion! I've updated the code to use a typed constexpr variable for better type safety.